xml/depl_maintenance.xml

```diff
@@ -508,6 +508,262 @@
+     Before you proceed, make sure that the database cluster has an odd number
+     of nodes. If necessary, extend the cluster as described in <xref linkend="sec.postgre-mariadb.upgrade.scenario2"/>. Install the latest maintenance updates on &productname; 7. In the
+     &crow; &wi;, switch to the <guimenu>Database</guimenu> &barcl;. You should
+     see the new <systemitem>mysql-server</systemitem> role in the
+     <guimenu>Deployment</guimenu> section. Add your &dbnode; to the
+     <systemitem>mysql-server</systemitem> role and apply the
+     &barcl;. &mariadb; is now deployed and running, but it is still not used
+     as a back end for &ostack; services. Do not change the
+     <literal>sql_engine</literal> at this point.
+    </para>
+    <!-- FIXME: Currently doesn't work accoring to devs
+	 <para>
+     If &rabbit; is using DRBD, enable the <guimenu>Use RabbitMQ
+     Clustering</guimenu> option in the &rabbit; &barcl; under <guimenu>High Availability</guimenu>. Since DRBD is not available
+     in &productname; 8, and &rabbit; clustering is supported in &productname;
+     7, you should switch to this option now.
+    </para> -->
+    <para>
+     Follow <xref linkend="postgre.mariadb.data.migration"/> to migrate the data from &postgres; to &mariadb;.
+    </para>
+    <procedure xml:id="postgre.mariadb.data.migration">
+     <title>Data Migration</title>
+     <step>
+      <para>
+           Run the <systemitem>/opt/dell/bin/prepare-mariadb</systemitem>
+	   script on the &admnode; to prepare the &mariadb; instance by creating
+	   the required users, databases, and tables.
+      </para>
+     </step>
+     <step>
+      <para>
+       After the script is finished, you'll find a list of all databases and
+       URLs that are ready for data migration in the
+       <filename>/etc/pg2mysql/databases.yaml</filename>. The script's output
+      may look as follows:</para>
+      <screen>Preparing node d52-54-77-77-01-01.vo6.cloud.suse.de
+Adding recipe[database::pg2mariadb_preparation] to run_list
+Running chef-client on d52-54-77-77-01-01.vo6.cloud.suse.de...
+Log: /var/log/crowbar/db-prepare.chef-client.log on
+d52-54-77-77-01-01.vo6.cloud.suse.de Run time: 444.725193199s
+Removing recipe[database::pg2mariadb_preparation] from run_list
+Prepare completed for d52-54-77-77-01-01.vo6.cloud.suse.de
+Summary of used databases: /etc/pg2mysql/databases.yaml on
+d52-54-77-77-01-01.vo6.cloud.suse.de</screen>
+      <para>The <literal>Summary of used databases:</literal> line shows the
+      exact location of the <filename>/etc/pg2mysql/databases.yaml</filename> file.
+      </para>
+      <para>
+       The <filename>/etc/pg2mysql/databases.yaml</filename> file contains a
+       list of databases along with their source and target connection strings:
+      </para>
+<screen>keystone:
+  source: postgresql://keystone:vZn3nfxXzv97@192.168.243.87/keystone
+  target:
+mysql+pymysql://keystone:vZn3nfxXzv97@192.168.243.88/keystone?charset=utf8
```

Comment:
This seems somewhat badly formatted. Could you move the "mysql+pymysql" line right after `target: ` ?

And if not, consider just using some fake password to make the strings shorted.

Reply:
Fixed.